### ISSUE TITLE
Missing styling on forum channels

### ISSUE THREAD

**Issue by fowlis** [catppuccin/discord#31]
__Issue:__
Background is missing on forum channels
![](https://cdn.upload.systems/uploads/BaHQc4Sn.png)
__Suggested fix:__
The class doesn't seem to be used anywhere else, and the colour is just my suggestion
```css
.list-3FIpnh {
    background-color: #1a1826;
}
```
__Preview of fix:__
![](https://cdn.upload.systems/uploads/gCa30Glz.png)
Sorry about the images being so big, I'm not sure how to scale them on Github

**Comment by TGRush** [comment]
I would actually match them a little less explicitly for future proofing:
```css
HTML-Element-Here[class^="list"] {
    background-color: #1a1826;
}
```

**Comment by nekowinston** [comment]
Closed with 2414d77b9a36dda361b6bc0bef3a3eb0e87f81a5
